addons/website/static/src/builder/plugins/options/dynamic_snippet_option.xml

```diff
@@ -51,11 +51,11 @@
         </BuilderButtonGroup>
     </BuilderRow>
     <BuilderRow id="number_of_records_opt_row" label.translate="Fetched Elements" t-if="!!dynamicOptionParams.domState.filterId or isSingleMode">
-        <BuilderSelect action="'numberOfRecords'" preview="false" id="'number_of_records_opt'">
+        <BuilderInputSelect action="'numberOfRecords'" preview="false" id="'number_of_records_opt'" default="1" min="1" type="'number'">
             <t t-foreach="['1','2','3','4','5','6','7','8','9','10','11','12','13','14','15','16']" t-as="value" t-key="value">
                 <BuilderSelectItem actionParam="value" t-out="value"/>
             </t>
-        </BuilderSelect>
+        </BuilderInputSelect>
```

Comment:
This option is applied on all dynamic snippets where we have filters but we have implemented Search more feature for Carousel snippet, what about other like blog?
<img width="1899" height="648" alt="image" src="https://github.com/user-attachments/assets/ef5d4af5-3a77-4b84-9b95-dd5438cc4b9f" />